### PR TITLE
[CLI] Add "ti cache clean" command to clean the offline cache files manually

### PR DESCRIPTION
**Pull request by PGZXB**
Issue: #6919

### CODE REVIEW

File: python/taichi/_main.py [hunk 1]
```diff
@@ -826,6 +827,28 @@ def lint(arguments: list = sys.argv[2:]):
         import pylint  # pylint: disable=C0415
         pylint.lint.Run(options)
 
+    @staticmethod
+    @register
+    def ticache(arguments: list = sys.argv[2:]):
```
Comment (lin-hitonami) [inline python/taichi/_main.py:832]: Will `ti cache clean` be better? Isn't the `ti` in `ticache` a bit redundant?

File: python/taichi/_main.py [hunk 2]
```diff
@@ -826,6 +827,28 @@ def lint(arguments: list = sys.argv[2:]):
         import pylint  # pylint: disable=C0415
         pylint.lint.Run(options)
 
+    @staticmethod
+    @register
+    def cache(arguments: list = sys.argv[2:]):
+        """Manage the offline cache files manually"""
+        if len(arguments) < 1:
```
Comment (lin-hitonami) [inline python/taichi/_main.py:834]:
```suggestion
        if not arguments:
```

File: python/taichi/_main.py [hunk 3]
```diff
@@ -826,6 +827,28 @@ def lint(arguments: list = sys.argv[2:]):
...
+            return
```
Comment (lin-hitonami) [inline python/taichi/_main.py:835]: Should we add an error message here?

**Review by lin-hitonami — APPROVED** [review]
LGTM! some nits:

**Comment by PGZXB** [comment]
> Should we add an error message here?

Thanks for your suggestion ~
I added some hint messages.

* `ti cache`
```
[Taichi] version 1.4.0, llvm 15.0.1, commit 02cce8c6, win, python 3.7.0
[I 12/21/22 21:06:27.015 70128] [shell.py:_shell_pop_print@23] Graphical python shell detected, using wrapped sys.stdout

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://docs.taichi-lang.org/
GitHub: https://github.com/taichi-dev/taichi/
Forum:  https://forum.taichi.graphics/

usage: ti cache <command> [<args>]
        clean   |-> Clean all offline cache files in given path
>>> Running time: 0.00s
```
* `ti cache faker`
```
[Taichi] version 1.4.0, llvm 15.0.1, commit 02cce8c6, win, python 3.7.0
[I 12/21/22 21:07:32.127 67672] [shell.py:_shell_pop_print@23] Graphical python shell detected, using wrapped sys.stdout

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://docs.taichi-lang.org/
GitHub: https://github.com/taichi-dev/taichi/
Forum:  https://forum.taichi.graphics/

'ti cache faker' is not a valid command!
usage: ti cache <command> [<args>]
        clean   |-> Clean all offline cache files in given path
>>> Running time: 0.00s
```
* `ti cache clean`
```
[Taichi] version 1.4.0, llvm 15.0.1, commit 02cce8c6, win, python 3.7.0
[I 12/21/22 21:10:11.792 36964] [shell.py:_shell_pop_print@23] Graphical python shell detected, using wrapped sys.stdout

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://docs.taichi-lang.org/
GitHub: https://github.com/taichi-dev/taichi/
Forum:  https://forum.taichi.graphics/

Deleted 9841 offline cache files in C:\taichi_cache\ticache
>>> Running time: 1.87s
```
* `ti cache clean faker`
```
[Taichi] version 1.4.0, llvm 15.0.1, commit 02cce8c6, win, python 3.7.0
[I 12/21/22 21:12:10.007 70904] [shell.py:_shell_pop_print@23] Graphical python shell detected, using wrapped sys.stdout

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://docs.taichi-lang.org/
GitHub: https://github.com/taichi-dev/taichi/
Forum:  https://forum.taichi.graphics/

usage: ti cache clean [-h] [-p OFFLINE_CACHE_FILE_PATH]
ti cache clean: error: unrecognized arguments: faker
```
* `ti cache clean -p ./2022-12-21-JOC/gfx`
```
[Taichi] version 1.4.0, llvm 15.0.1, commit 02cce8c6, win, python 3.7.0
[I 12/21/22 21:13:17.783 33348] [shell.py:_shell_pop_print@23] Graphical python shell detected, using wrapped sys.stdout

*******************************************
**      Taichi Programming Language      **
*******************************************

Docs:   https://docs.taichi-lang.org/
GitHub: https://github.com/taichi-dev/taichi/
Forum:  https://forum.taichi.graphics/

Deleted 15 offline cache files in D:\CODE\CPP_CODE\taichi_test\2022-12-21-JOC\gfx
>>> Running time: 0.00s
```

**Review by lin-hitonami — APPROVED** [review]
LGTM

**Pull request merged**